api/BaseAudioContext.json

```diff
@@ -5,54 +5,40 @@
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/BaseAudioContext",
         "support": {
           "chrome": {
-            "version_added": true
+            "version_added": "14"
```

Comment:
This isn't from mdn-bcd-collector, right? Can you update the data to match the collector results, and not any manual changes from that? `BaseAudioContext` was introduced much later than this.

Reply:
Unfortunately, this will cause consistency issues if we set it to what comes from the collector, which is "56".  We've got entries within here that were initially implemented on just AudioContext.